kore/app/format/Main.hs

```diff
@@ -4,80 +4,84 @@ import Prelude.Kore
 
 import qualified Data.Text.IO as Text
 import Options.Applicative
-import System.IO
-    ( stdout
-    )
+import System.IO (
+    stdout,
+ )
 
-import Kore.Parser
-    ( parseKoreDefinition
-    )
-import Kore.Syntax.Definition
-    ( ParsedDefinition
-    )
+import Kore.Parser (
+    parseKoreDefinition,
+ )
+import Kore.Syntax.Definition (
+    ParsedDefinition,
+ )
 import Kore.Unparser
-import Pretty
-    ( LayoutOptions (..)
-    , PageWidth (..)
-    , defaultLayoutOptions
-    , layoutPretty
-    , renderIO
-    )
+import Pretty (
+    LayoutOptions (..),
+    PageWidth (..),
+    defaultLayoutOptions,
+    layoutPretty,
+    renderIO,
+ )
 
 import GlobalMain
 
-
-data KoreFormatOptions =
-    KoreFormatOptions
-    { fileName :: FilePath  -- ^ file to unparse
-    , width    :: Int  -- ^ line width
+data KoreFormatOptions = KoreFormatOptions
+    { -- | file to unparse
+      fileName :: FilePath
+    , -- | line width
+      width :: Int
     }
 
 commandLine :: Parser KoreFormatOptions
 commandLine =
     KoreFormatOptions
-    <$> argument str
-        (  metavar "FILE"
-        <> help "Kore source file to parse" )
-    <*> option auto
-        (  metavar "WIDTH"
-        <> long "width"
-        <> value 80
-        <> help "Line width [default: 80; unlimited if WIDTH <= 0]" )
+        <$> argument
+            str
+            ( metavar "FILE"
+                <> help "Kore source file to parse"
+            )
+        <*> option
+            auto
+            ( metavar "WIDTH"
+                <> long "width"
+                <> value 80
+                <> help "Line width [default: 80; unlimited if WIDTH <= 0]"
+            )
```

Comment:
I'm not thrilled with the extra indentation that fourmolu introduces here. I don't think this makes the code more readable, and it reduces the available horizontal space. On the other hand, reducing the available space is going to force us to extract more definitions, which is good.